esp/eclwatch/ws_XSLT/WUQueryDetails.xslt

```diff
@@ -141,7 +141,7 @@
                               }
                           };
 
-                          var postBody = '<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:SOAP-ENC="http://schemas.xmlsoap.org/soap/encoding/" xmlns="http://webservices.seisint.com/ws_roxieconfig"><soap:Body>' + getAliasActions(Action) + '</soap:Body></soap:Envelope>';
+                          var postBody = '<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:SOAP-ENC="http://schemas.xmlsoap.org/soap/encoding/" xmlns="http://webservices.seisint.com/ws_workunits"><soap:Body>' + getAliasActions(Action) + '</soap:Body></soap:Envelope>';
```

Comment:
This probably won't have any affect, but for WsWorkunits this was: http://webservices.seisint.com/WsWorkunits.

Changing it to that will make it consistent with elsewhere in this file, and similar files.

Modern WsWorkunits actually has a completely different string, but I doubt it matters and updating the rest of these strings isn't in the scope of this PR.